config.yaml

```diff
@@ -70,3 +70,13 @@ options:
       The private docker registry configured as dockerhub mirror to be used by the runners. If set 
       a message will be displayed prior to job execution on self-hosted runner informing users to 
       use the provided registry.
+  aproxy-address:
+    type: string
+    default: ""
+    description: |
+      (Experimental, may be removed) When set to a http proxy server socket address 
+      (format hostname:port, e.g. squid.internal:3128),
```

Comment:
The nftables rules we used is a generic one, which means that the proxy ip address can't not be a public ip with proxy port is 80 or 443 at the same time. Or else, the nftables rule will cause an infinite loop. Maybe we should include that in the document?

Reply:
Shouldn't we just use the model proxy configuration for this?

Reply:
Originally I thought it would be better to be independent of the juju proxy configuration, since aproxy was installed in the charm and could be used independently. With the current approach, we still need the model proxy configuration for the charm, so it makes sense to reuse that setting for aproxy. I have updated the code.